src/OSWindow-SDL2/SDLNullPlatform.class.st

```diff
@@ -0,0 +1,18 @@
+"
+I am the default platform. All my methods are empty or have a default implementation
+"
+Class {
+	#name : #SDLNullPlatform,
+	#superclass : #SDLAbstractPlatform,
+	#category : #'OSWindow-SDL2-Bindings'
+}
+
+{ #category : #operations }
+SDLNullPlatform >> afterSetWindowTitle: aString onWindow: aOSSDLWindow [
```

Comment:
Why is it called `after*` ?

Reply:
Because it is invoked after changing the window title in SDL.